packages/resource-deployment/scripts/delete-resource-group.sh

```diff
@@ -48,11 +48,34 @@ deleteResourceGroup() {
 }
 
 deleteApimIfExists() {
-    response=$(az apim list --resource-group $resourceGroupName --query "[?name=='$apiManagementName']" -o tsv)
+    apimExistsCommand="az apim list --resource-group $resourceGroupName --query \"[?name=='$apiManagementName']\" -o tsv"
+    apimExists=$(eval "$apimExistsCommand")
```

Comment:
do we need this eval? Will the old way work instead: `apimExistsCommand=$(az apim ...)`